assets/js/main_albers.js

```diff
@@ -612,22 +613,37 @@ map.on('load', function() {
         false 
     ])
 
+    // What's returned is an object, and we need an array of objects.
+    const flattenDistrictDataIntoGeoJSON = function(obj) {
+        var features = [];
+        // In order to get the length of an object, we need to get the keys, then get their length - we
+        // can't just get it directly.
+        for (var i = 0; i < Object.keys(obj).length; ++i) {
+            features.push(obj[i][0]);
+        }
+        return features;
+    }
+
     map.addSource('state_districts',{
         type: 'geojson',
         data: {
                 'type': 'FeatureCollection',
-                'features': district_data.map(district => getDistrictShapes(district, true)).flat()
+                'features': flattenDistrictDataIntoGeoJSON(
```

Comment:
It seems like these produce the same output of [featureObject, featureObject], e.g. 
![image](https://user-images.githubusercontent.com/938890/88099894-abe5ae80-cb50-11ea-80f8-cf6e7734177a.png).

Is this method better because it's more general?